src/image.rs

```diff
@@ -855,23 +855,23 @@ impl<'a> BuildImageQueryParams<&'a str> for BuildImageOptions<String> {
 #[serde(deny_unknown_fields)]
```

Comment:
You'll need to add `#[allow(missing_docs)]` to all the structs that now have public methods to get it to compile.

Reply:
Thanks for the comment. I pushed a new commit to fix this.